core/modules/widgets/checkbox.js

```diff
@@ -3,167 +3,118 @@ title: $:/core/modules/widgets/checkbox.js
 type: application/javascript
 module-type: widget
 
-Checkbox widget
+The Checkbox widget toggles the value of a field between two string values
 
 \*/
-(function(){
+(function () {
 
-/*jslint node: true, browser: true */
-/*global $tw: false */
-"use strict";
+	/*jslint node: true, browser: true */
+	/*global $tw: false */
```

Comment:
No indent here. Have a look at the core code. ... the top function never causes the code to be indented, because its only a wrapper. ..